packages/component-base/src/polylit-mixin.js

```diff
@@ -105,7 +106,7 @@ const PolylitMixinImplementation = (superclass) => {
             this.requestUpdate(name, oldValue, options);
 
             // Enforce synchronous update
-            if (this.hasUpdated) {
+            if (notEqual(oldValue, value) && this.hasUpdated) {
```

Comment:
**note:** Lit supports customizing property change detection with hasChanged. However, I decided that adding it to PolylitMixin is not useful since Polymer doesn't support this feature anyway.